homeassistant/components/minecraft_server/__init__.py

```diff
@@ -39,7 +39,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         )
     except MinecraftServerAddressError as error:
         raise ConfigEntryError(
-            f"Server address in configuration entry is invalid (error: {error})"
+            f"Server address in configuration entry is invalid ({repr(error)}: {error})"
```

Comment:
I don't know the reason, but for some errors I don't get any error message. But with `repr(error)` I at least get the error type. During testing I didn't see any logs, where both, error type and error message were shown. If no message is available, it looks for example like this: `TimeoutError(): `

Reply:
I'm not sure I see how this could happen. This is a  MinecraftServerAddressErro (not a TimeoutError?) and its always thrown with an explicit error message from what I can tell: https://github.com/home-assistant/core/blob/b4295e909ce96fb61581f333520771c51195f84f/homeassistant/components/minecraft_server/api.py#L64

Reply:
Correct, but with the addition of the error message from the library, which sometimes is empty, for whatever reason.

Reply:
So then what I would say is if the message argument to `MinecraftServerAddressError` is not correct, then update that to be valid.

Reply:
Or another proposal: For me it would be sufficient to have only `repr(error)`. Is this also fine for you?